vsintegration/src/vs/FsPkgs/FSharp.LanguageService/FSharp.LanguageService.Base/CodeWindowManager.cs

```diff
@@ -14,7 +14,6 @@
 using Microsoft.VisualStudio.FSharp.LanguageService.Resources;
 
 namespace Microsoft.VisualStudio.FSharp.LanguageService {
-    /// <include file='doc\CodeWindowManager.uex' path='docs/doc[@for="CodeWindowManager"]/*' />
```

Comment:
i splitted changes in multiple commits :smile: , feel free to use what you think is ok.
I understand the uex include is good for reference doc/examples etc, so np if used, but i dont see this repo on msdn